ADALiOS/ADALiOS/ADAuthenticationBroker.m

```diff
@@ -251,14 +218,13 @@ - (void)start:(NSURL *)startURL
     }
     else
     {
-        if (!parent)
-        {
-            // Must have a parent view controller to start the authentication view
-            parent = [UIApplication adCurrentViewController];
-        }
-        
-        if (parent)
+        _windowController = [[ADWindowController alloc] init];
+        if (_windowController)
         {
+#if TARGET_OS_IPHONE
+            [_windowController setParentController:parent];
+            [_windowController setFullScreen:fullScreen];
+#endif // TARGET_OS_IPHONE
             _ntlmSession = [ADNTLMHandler startWebViewNTLMHandlerWithError:nil];
```

Comment:
The interaction between the NTLM Session Handler and launching the window is not as clean as it should be. I'm going to add a notification here to better scope when that is happening.